Revise CrowdStrike installation guide formatting

nonpunctual opened the pull request:
Updated formatting and links for CrowdStrike installation instructions, ensuring consistency in naming and image references.

nonpunctual committed:
Revise CrowdStrike installation guide formatting

Updated formatting and links for CrowdStrike installation instructions, ensuring consistency in naming and image references.

nonpunctual committed:
Fix links and filenames in CrowdStrike deployment guide

Updated links and filenames to use lowercase for consistency.

nonpunctual committed:
Fix image path and correct CrowdStrike directory name

nonpunctual committed:
Correct payload names in CrowdStrike documentation

Updated payload names for CrowdStrike Falcon configuration.

nonpunctual committed:
Fix image file extension in CrowdStrike deployment guide

Updated image file extension for configuration profiles section.

nonpunctual committed:
Fix minor wording and link formatting issues

iansltx reviewed (commented):
Assuming the blockquotes render correctly (did _not_ pull this branch down to check), LGTM from a content accuracy perspective. Just need to fix the extra space issue, which I'll do now so I can approve this.

iansltx committed:
Remove extra space

mike-j-thomas committed:
Update deploying-crowdstrike-with-fleet.md

- Converted the note blocks to regular body content, since these sections are the main content rather than callouts.
- Kept the heading hierarchy consistent (H3s under H2s).
- Normalized headings to use “CrowdStrike Falcon” for clarity and searchability.
- Avoided using “CrowdStrike” on its own when referring to the Falcon product.

mike-j-thomas reviewed (approved):
Thanks, @nonpunctual, I made a few changes to keep things consistent:

- Converted the note blocks to regular body content, since these sections are the main content rather than callouts.
- Kept the heading hierarchy consistent (H3s under H2s).
- Normalized headings to use “CrowdStrike Falcon” for clarity and searchability.
- Avoided using “CrowdStrike” on its own when referring to the Falcon product.

mike-j-thomas commented:
@nonpunctual , if you want to keep those install options as easy-to-scan callouts, you could put them directly under the intro with the subheadings built into each note.

Callouts under intro
<img width="1588" height="1100" alt="Screenshot 2025-12-16 at 9 16 38" src="https://github.com/user-attachments/assets/136e7005-7b56-459c-8f1b-9689734a672a" />

No callouts (not as clear if you intentionally wanted them to be highlighted:
<img width="1594" height="1245" alt="Screenshot 2025-12-16 at 9 13 57" src="https://github.com/user-attachments/assets/e5db64cf-0ed0-404d-b0de-c4d25fd56735" />

Callouts under subheadings (feels off since the callouts are the main content under these subheadings)
<img width="1588" height="1225" alt="Screenshot 2025-12-16 at 9 14 09" src="https://github.com/user-attachments/assets/a5f64c3f-54d8-44e0-8bee-63ac2eb115c8" />

nonpunctual commented:
either way! @mike-j-thomas
